bitcrowd/bitstyles
# [#711] Update button variants

Fixes #711 

## Changes

- Adds new Poppins webfonts
- Size & colors for default-theme primary, secondary, outline, transparent, and tab buttons
- Button border-width is now automatically subtracted from the padding, so the height of all buttons is the same regardless of border-width (like having `inside` borders, which is not possible with CSS otherwise)
- Adds storybook-zeplin and links every button-color story to a component in the styleguide
 
🚓 Updates every mdx file to import storybook docs block in the newer non-deprecated way

## 📸

<img width="1028" alt="Screenshot 2023-03-29 at 11 07 29" src="https://user-images.githubusercontent.com/2479422/228484979-6459f2a2-5a31-4f96-838c-a4ecdbb17c5a.png">

## How to QA Visual changes

- `git checkout main` 
- `git pull` 
- `git checkout feature/711-update-button-variants` 
- `yarn`
- `yarn storybook`
- navigate to button component (see the “overview” page at the bottom of the stories)

Check:

- [ ] The appearance matches the zeplin designs (using the storybook Zeplin plugin — please let me know if I made any mistakes with the spline links! There are a lot of them)

## How to QA code changes


## Preflight checks

PR author to check, and delete if not applicable:
